src_Core/CPU/RISCV_MBox.bsv

```diff
@@ -214,6 +215,7 @@ module mkRISCV_MBox (RISCV_MBox_IFC);
       // MUL, MULH, MULHU, MULHSU
       if (f3 [2] == 1'b0) begin
 	 rg_state <= STATE_MUL1;
+	 rg_valid <= False;
```

Comment:
You also need `rg_valid <= False` near line 257 where `rg_state` is set ts`STATE_DIV_REM`. (Github wouldn't let me put the comment on that line because it was too far away from your changes.)

Reply:
also fixed. I thought I was searching for all the updates of rg_state but clearly missed that.